jujugui/static/gui/src/app/assets/css/_mixins.scss

```diff
@@ -97,11 +98,44 @@
     color: $charm-panel-orange;
     text-align: center;
 
+    &:active,
+    &:focus,
     &:hover {
         background-color: $light-grey;
     }
 }
 
+%button-orange {
+    @extend %button;
+    border: none;
+    background-color: $ubuntu-orange;
+    color: $white;
+
+    &:active,
+    &:focus,
+    &:hover {
+        $colour: #ae3910;
```

Comment:
I think it would be easier to maintain if you used `darken($ubuntu-orange, 10%)`